config/samples/config.yaml

```diff
@@ -15,16 +15,3 @@ leader_election:
   retry_period: 2s                      # retry_period is the time in seconds that the acting controller
                                         # will wait between tries of actions with the controller.
   disable: false                        # Whether to disable leader election.
-
```

Comment:
log_level: "info"                               # The log level of the API7 Ingress Controller.